tests/cxx/subfolder/direct_subfolder.h

```diff
@@ -0,0 +1,19 @@
+//===--- direct_subfolder.h - test input file for iwyu --------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// This file includes and uses only another file in the subfolder
+
+#ifndef DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_TESTS_DIRECT_SUBFOLDER_H_
+#define DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_TESTS_DIRECT_SUBFOLDER_H_
+
+#include "indirect_subfolder.h"
+
+namespace { IndirectSubfolderClass foo; }
```

Comment:
The comment is already there: `// This file includes and **uses** only another file in the subfolder`
Note that this is the common way in the IWYU tests for indirect includes: Include a header that uses the used type but do not include the types header itself. So yes this single line is not required for the checks. But it IS required for the standard 'indirect' model used in the tests.